Cargo.toml

```diff
@@ -8,8 +8,10 @@ default-members = [
   "crates/bitmex-stream",
   "crates/coordinator-commons",
   "crates/ln-dlc-node",
+  # "crates/ln-dlc-node-tests",
   "crates/orderbook-client",
   "crates/orderbook-commons",
+  # "crates/tests-e2e",
```

Comment:
we are not statically linking them in our binaries.

this means faster compile time (less things to link) and space saving. 

of course leaving them commented out is not needed, I just wanted to be explicit about these exclusions to draw attention to that 🙃

Reply:
> we are not statically linking them in our binaries.
>
> this means faster compile time (less things to link) and space saving.

Wait, really? Just because we declare them in the workspace we are linking test crates into our binaries? That doesn't seem right since they should not be in the dependency tree of the binaries 🤔